### PR TITLE
potatoCode first blogpost

### PR DESCRIPTION
**Pull request by aleuca**
We've finished our first blog post!

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -125,9 +125,12 @@ PLATFORMS
 
 DEPENDENCIES
```
Comment (v-thakkar) [inline Gemfile.lock]: HI @aleuca ! Thanks for the PR. Please send one change per one PR. If you wish to send changes for the dependencies file, please send it in the separate PR and keep this PR for the blog post.

File: blog/_posts/2018-07-04-first-official-cintribution-team-potatoCode.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+title: First Week Wrap Up! Team potatoCode
+layout: post
+created_at: Sat Jul 07 2018
+permalink: blog/2018-06-28-first-official-cintribution-team-potatocode
```
Comment (v-thakkar) [inline blog/_posts/2018-07-04-first-official-cintribution-team-potatoCode.md]: Permalink should be same as file name. Dates are different here.

**Review by v-thakkar — CHANGES_REQUESTED** [review]

**Comment by aleuca** [comment]
@nerdyvaishali All set!

File: blog/_posts/2018-07-07-first-official-contribution-team-potatoCode.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+We had our first call with our mentor Jan on Wednesday, and he is wonderful! He gave us ideas for first issues and functionalities, and has already helped us figure out how to contribute to the project. We're excited for a potential visit from him some time during the program!
+
+We also had our first call with our supervisor Ana Sofia and had a blast! Besides discussing all the oportunities we'll have to meet up in Portugal or Amsterdam, she also helped us a TON to feel welcomed into the program.
+
+On Saturday we got together to work on Stretchly again, and were lucky to have our coach Dan with us the entire day. We managed to get a lot of work done, and especially decipher the app a lot more with his help.
+
+We completed our first little feature -- translation of the app into Dutch, and our first pull request!
+
+<br>
+<span class="color-red">We had our first dinner with all the coaches!</span>
```
Comment (v-thakkar) [inline blog/_posts/2018-07-07-first-official-contribution-team-potatoCode.md]: Not sure what was your intention behind using span class here [and below] but it seems that this tag doesn't do anything at the moment. May be span style?

File: blog/_posts/2018-07-07-first-official-contribution-team-potatoCode.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
+---
+title: First Week Wrap Up! Team potatoCode
+layout: post
+created_at: Sat Jul 07 2018
+permalink: 2018-07-07-first-official-contribution-team-potatoCode
```
Comment (v-thakkar) [inline blog/_posts/2018-07-07-first-official-contribution-team-potatoCode.md]: Well, the format for permalink is blog/<file_name>. What I meant was to just change the date. Sorry if that wasn't clear.

**Review by v-thakkar — CHANGES_REQUESTED** [review]

File: blog/_posts/2018-07-07-first-official-contribution-team-potatoCode.md (hunk 3 above)
Comment (aleuca) [reply blog/_posts/2018-07-07-first-official-contribution-team-potatoCode.md]: I thought there was css modifying this span? To me it shows up as red.

Comment (v-thakkar) [reply blog/_posts/2018-07-07-first-official-contribution-team-potatoCode.md]: Sorry my bad. Looks good to me. Thanks.

File: blog/_posts/2018-07-07-first-official-contribution-team-potatoCode.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+We completed our first little feature -- translation of the app into Dutch, and our first pull request!
+
+<br>
+<span class="color-red">We had our first dinner with all the coaches!</span>
+<br>
+
+This was a huge highlight of our week! It was Sabine's first time meeting Eden, and both of our first time meeting Jack. We got to know each other a bit, and it helps us feeling more comfortable with our coaches. We can only hope it makes the coaches more comfortable with us as well! We hope to do this again (maybe once a month?) during the program.
+
+![image](/img/blog/2018/2018-07-05-potatocode-team.jpg)
+<div class="image-credits">potatoCode team! (Photo: Sabine van der Eijk, Daniel Rigberg, Jack Sacali, Eden Laboda, Alina Leuca)</div>
```
Comment (v-thakkar) [inline blog/_posts/2018-07-07-first-official-contribution-team-potatoCode.md]: @aleuca Sorry, I missed this in my last review but this seems like just the names of people in the photo. Can you add photo credits, something like, 'photo by [device/person]'? We're good to go after that minor change. Thanks.

**Review by v-thakkar — CHANGES_REQUESTED** [review]

**Comment by aleuca** [comment]
Should be good to go now, hopefully! :)

**Review by v-thakkar — APPROVED** [review]

**Comment by v-thakkar** [comment]
Thanks for the quick fix. :)
